twoliter/src/project.rs

```diff
@@ -156,6 +193,23 @@ impl Project {
     }
 }
 
+/// This represents a container registry vendor that is used in resolving the kits and also
+/// now the bottlerocket sdk
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq, Ord, PartialOrd)]
+#[serde(rename_all = "kebab-case")]
+pub(crate) struct Vendor {
+    pub registry: String,
+}
+
+/// This represents a dependency on a container, primarily used for kits
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq, Ord, PartialOrd)]
+#[serde(rename_all = "kebab-case")]
+pub(crate) struct Dependency {
```

Comment:
I don't like the name `Dependency`. It is over-used already.

```suggestion
pub(crate) struct Image {
```

Reply:
Fair call on `Dependency` but `Image` is perhaps the most overloaded word we have across Bottlerocket 😆